config/crd/kustomizeconfig.yaml

```diff
@@ -5,12 +5,12 @@ nameReference:
   fieldSpecs:
   - kind: CustomResourceDefinition
     group: apiextensions.k8s.io
-    path: spec/conversion/webhookClientConfig/service/name
+    path: spec/conversion/webhook/clientConfig/service/name
 
 namespace:
 - kind: CustomResourceDefinition
   group: apiextensions.k8s.io
-  path: spec/conversion/webhookClientConfig/service/namespace
+  path: spec/conversion/webhook/clientConfig/service/namespace
   create: false
 
 varReference:
```

Comment:
Why is this changing at all?

Reply:
my kustomize isn't the best... but this is the patch:
```
apiVersion: apiextensions.k8s.io/v1
kind: CustomResourceDefinition
metadata:
  name: opentelemetrycollectors.opentelemetry.io
spec:
  conversion:
    strategy: Webhook
    webhook:
      clientConfig:
        service:
          namespace: system
          name: opentelemetry-operator-webhook-service
          path: /convert
```

When i tried to install, i got this error:
```
The CustomResourceDefinition "opentelemetrycollectors.opentelemetry.io" is invalid: spec.conversion.webhookClientConfig: Required value: exactly one of url or service is required
```

Reply:
Isn't this file autogenerated by kubebuilder? If not, where did it come from?

Reply:
FYI, the rest of the PR looks good to me, I'm just confused about this specific file.

Reply:
i don't think this is autogenerated... cc @pavolloffay

Reply:
it should be auto-generated, I am not sure why it is changed in this PR

Reply:
this was preventing my local instance from starting... the syntax looked wrong to me... i can revert this and maybe i need to update my local kustomize or something